### PR TITLE
fix(index): prevent crashing from undefined errors

### PR DESCRIPTION
**Pull request by trieloff**
see https://dashboard.epsagon.com/search/?search_filters%5B0%5D%5Bfilter%5D=resource_name&search_filters%5B0%5D%5Bop%5D=%3D&search_filters%5B0%5D%5Bvalue%5D=content-proxy&search_filters%5B1%5D%5Bfilter%5D=error&search_filters%5B1%5D%5Bop%5D=%3D&search_filters%5B1%5D%5Bvalue%5D=true&search_filters%5B2%5D%5Bfilter%5D=exception.type&search_filters%5B2%5D%5Bop%5D=%3D&search_filters%5B2%5D%5Bvalue%5D=TypeError&time_frame%5Btype%5D=custom&time_frame%5Bfrom%5D=1590396335169&time_frame%5Bto%5D=1590569135169&time_frame%5Bfrequency%5D=86400 for example occurences

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -118,9 +118,13 @@ async function main({
         body: e.message,
       };
     }
-    log.error('Unhandled error', e, e.stack);
+    log.error('Unhandled error', e,
+    /* istanbul ignore next */
+      e.stack || 'no stack');
```
Comment (tripodsan) [inline src/index.js:123]: but this doesn't prevent this error:

```
Cannot read property 'stack' of undefined
```

should be:

```
(e && e.stack) || 'no stack'
```

File: src/index.js [hunk 2]
```diff
@@ -118,9 +118,13 @@ async function main({
...
     return {
-      body: e.message,
+      body:
+      /* istanbul ignore next */
+      e.message || 'no message',
```
Comment (tripodsan) [inline src/index.js:127]: same here. if `e` is undefined, this throws.

File: src/index.js [hunk 3]
```diff
@@ -118,9 +118,13 @@ async function main({
...
       statusCode:
       /* istanbul ignore next */
       e.status || 500,
```
Comment (tripodsan) [inline src/index.js:130]: same here.

File: src/index.js (hunk 1 above)
Comment (trieloff) [reply src/index.js:123]:
```suggestion
      (e && e.stack) || 'no stack');
```

File: src/index.js (hunk 2 above)
Comment (trieloff) [reply src/index.js:127]:
```suggestion
      (e && e.message) || 'no message',
```

File: src/index.js (hunk 3 above)
Comment (trieloff) [inline src/index.js:130]:
```suggestion
      (e && e.status) || 500,
```

**Pull request merged**